core/tests/webdriverio_desktop/publicationAndLibrary.js

```diff
@@ -190,10 +190,13 @@ describe('Library index page', function() {
     // Private explorations are not shown in the library.
     await libraryPage.expectExplorationToBeHidden('Vilya');
 
+    await libraryPage.selectLanguages([LANGUAGE_FRANCAI]);
     await libraryPage.findExploration(EXPLORATION_VINGILOT);
```

Comment:
English is not the language of any of the exploration card present here, it was useless to select English here as this was causing the flake. I just removed English from language selection and selected the actual language of the exploration card that is being searched.